.env

```
@@ -0,0 +1,8 @@
+keycloak_admin_user=admin
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The keycloak_admin_user key should be in uppercase

(LowercaseKey)